djangomain/urls.py

```diff
@@ -52,7 +52,7 @@
     path("api/token/refresh/", TokenRefreshView.as_view(), name="token_refresh"),
     path("api/token/verify/", TokenVerifyView.as_view(), name="token_verify"),
     path("measurement/", include("measurement.urls", namespace="measurement")),
-    path("importing/", include("importing.urls", namespace="importing")),
+    # path("importing/", include("importing.urls", namespace="importing")),
```

Comment:
Why is this commented out

Reply:
For no reason. I've deleted it, as it was not in use, anyway.